### ISSUE TITLE
Migrate captcha solver from Python to ONNX Runtime in Node.js

### ISSUE THREAD

**Issue by ktenman** [ktenman/portfolio#1118]
## Summary

Migrate the captcha solver from Python FastAPI + TensorFlow to ONNX Runtime running directly in Node.js cloudflare-bypass-proxy.

## Current Architecture

```
Auto24 request → cloudflare-bypass-proxy → HTTP call → Python FastAPI → TensorFlow inference
                                         ~500ms overhead
```

## Target Architecture

```
Auto24 request → cloudflare-bypass-proxy → ONNX Runtime inference (in-process)
                                         ~30ms
```

## Expected Improvement

- **90% faster** captcha solving (500ms → 30ms)
- Eliminates HTTP latency
- Eliminates Python process overhead
- Simpler deployment (one less container)

## Tasks

- [ ] Convert `model.keras` to `model.onnx` format
- [ ] Add `onnxruntime-node` to cloudflare-bypass-proxy
- [ ] Implement TypeScript inference wrapper
- [ ] Update `auto24.ts` to use local inference
- [ ] Benchmark and validate accuracy
- [ ] Remove Python captcha-solver dependency

## Technical Details

Model specs:
- Input: `(1, 25, 65, 1)` - grayscale image
- Output: 4 character predictions (27 classes each)
- Characters: `2345789ABCDEFHKLMNPRTUVWXYZ`

## Conversion Script

```python
import tensorflow as tf
import tf2onnx

model = tf.keras.models.load_model('model.keras')
input_signature = [tf.TensorSpec(shape=(1, 25, 65, 1), dtype=tf.float32)]
tf2onnx.convert.from_keras(model, input_signature=input_signature, opset=13, output_path='model.onnx')
```

## Related

- Closes #1116 (partial - this is the captcha optimization portion)

**Comment by ktenman** [comment]
Completed in PR #1119. The captcha solver has been migrated from a separate Python/TensorFlow service to an embedded ONNX model in the cloudflare-bypass-proxy Node.js service.

**Key improvements:**
- Docker image size: ~2GB → ~150MB
- Inference speed: ~50ms → ~5ms
- No separate container needed
- Using `pngjs` for optimized PNG parsing
